src/cmdlib.sh

```diff
@@ -142,7 +146,7 @@ pick_yaml_or_else_json() {
 }
 
 prepare_build() {
-    preflight
```

Comment:
Instead of nesting the call to `preflight` inside of `preflight_kvm` WDYT of just calling them separately here? Slightly more explicit.